src/main/scala/com.snowplowanalytics/forex/oerclient/OerClient.scala

```diff
@@ -228,11 +228,7 @@ class OerClient(
           while (root.hasNext) {
            resNode = root.next
           }
-          if (resNode.getTextValue.contains("Invalid App ID")) {
-            Left(OerResponseError(resNode.getTextValue, InvalidAppId))
-          } else {
-            Left(OerResponseError(resNode.getTextValue, OtherErrors))
-          }
+          Left(OerResponseError(resNode.getTextValue, OtherErrors))
```

Comment:
This has been simplified since the API returns a usual 404 no matter if the app is valid or not.